Packs/AzureNetworkSecurityGroups/Integrations/AzureNetworkSecurityGroups/AzureNetworkSecurityGroups_description.md

```diff
@@ -1,54 +1,75 @@
 In order to connect to the Azure Network Security Groups use one of the following methods:
 
 1. *Authorization Code Flow* (Recommended).
+2. *Client Credentials Flow*
 2. *Device Code Flow*.
 3. *Azure Managed Identities Flow*.
 
+## Self-Deployed Azure App
+
+To use a self-configured Azure application, you need to add a new Azure App Registration in the Azure Portal.
+
+To add the registration, refer to the following [Microsoft article](https://learn.microsoft.com/en-us/defender-xdr/api-create-app-web?view=o365-worldwide) steps 1-8.
+
+The application must have *user_impersonation* and *offline_access* permissions and must allow public client flows (can be found under the **Authentication** section of the app).
```

Comment:
Let's change to: 
### Required permissions
- Azure Service Management - permission `user_impersonation` of type `Delegated`

* `and must allow public client flows (can be found under the **Authentication** section of the app).` - this part is only relevant for Device Code flow, you can delete it

Reply:
please update also in the README